### PR TITLE
Ml image index

### LINKED ISSUE #3: Image Index
#### As a user I want the homepage to display all saved images.

Acceptance criteria:
- [x] New images that are added show up on the homepage.
- [x] These images are persisted if the browser is closed or even if the
  server is restarted.
- [x] Images are not displayed wider than 400px.

Dependencies:
- Save Image Link

### PR DESCRIPTION
**Pull request by gitlev**
Closes #3

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -24,7 +24,5 @@ node_modules
 *c1*
 
 # Ignore coffee and scss
-*.coffee
-*.scss
-
-
+# *.coffee
```
Comment (dougthomas) [inline .gitignore]: It is rarely good form to include commented-out code.

File: app/models/image.rb [hunk 2]
```diff
@@ -1,3 +1,6 @@
 class Image < ActiveRecord::Base
+
   validates :url, :presence => true
+  scope :reverse_sorted_by_ids, -> { order(created_at: :desc) }
```
Comment (dougthomas) [inline app/models/image.rb]: This scope's name and definition are not in agreement.  Change one or the other.

File: test/controllers/images_controller_test.rb [hunk 3]
```diff
@@ -4,23 +4,33 @@ class ImagesControllerTest < ActionController::TestCase
...
 
+  #view test / integrated test
+  test 'create image added to index' do
+    Image.create!(url: 'https://i.kinja-img.com/gawker-media/image/upload/s--F5WL38pY--/c_scale,fl_progressive,q_80,w_800/yfcj1ufs4u3mofay9zyf.jpg')
+    get :index
+    assert_response :success
+    assert_select 'img' do |img_tags|
+      assert_equal 'https://i.kinja-img.com/gawker-media/image/upload/s--F5WL38pY--/c_scale,fl_progressive,q_80,w_800/yfcj1ufs4u3mofay9zyf.jpg', img_tags[0].attributes['src'].value
+    end
+    # assert_select('img','asdf')
```
Comment (dougthomas) [inline test/controllers/images_controller_test.rb]: Remove

**Comment by dougthomas** [comment]
I made some minor comments.  Once those are addressed, this is good to go!

File: .gitignore (hunk 1 above)
Comment (gitlev) [inline .gitignore]: ml - removed commented code from .gitignore
https://github.com/AppFolioOnboarding/image-sharer-gitlev/commit/859069335a8241c1bb70839873a88c912abb17e9

File: app/models/image.rb (hunk 2 above)
Comment (gitlev) [inline app/models/image.rb]: ml - changed name of scope to accurately describe definition
https://github.com/AppFolioOnboarding/image-sharer-gitlev/commit/3a9b644f899b4c333bb17fed89b42c3b13a7cb73

File: test/controllers/images_controller_test.rb (hunk 3 above)
Comment (gitlev) [inline test/controllers/images_controller_test.rb]: Why should this be removed?  (One of the criteria for Issue#3 is ' New images that are added show up on the homepage.')

Comment (dougthomas) [inline test/controllers/images_controller_test.rb]: Sorry; my comment was meant to indicate that you should remove the commented-out assertion.

Comment (gitlev) [inline test/controllers/images_controller_test.rb]: ml - removed comment from image_controller_test.rb
https://github.com/AppFolioOnboarding/image-sharer-gitlev/commit/d364598afa967f6a80b9cdae4b8203319157ebfa

**Pull request merged**